### ISSUE TITLE
Private chat 

### ISSUE THREAD

**Issue by nitin-787** [nitin-787/mynotes#49]
### Add the 1:1 chatting feature in the app

**Comment by Mehul-Kumar-27** [comment]
Hey , can I work on this issue ??

**Comment by Mehul-Kumar-27** [comment]
I would like to work on this issue.

**Comment by nitin-787** [comment]
> I would like to work on this issue.

![download](https://user-images.githubusercontent.com/76255199/193869437-e8404cca-653f-4359-8d98-89a87aaa8278.jpg)

**Comment by nitin-787** [comment]
## Chat main page ui
![chat_screen](https://user-images.githubusercontent.com/76255199/194333009-3ca0056a-69be-43dc-8645-e8b29a0d6163.png)

**Comment by nitin-787** [comment]
any progress @Mehul-Kumar-27 ?

**Comment by aashray374** [comment]
I would like to work on this issue.

**Comment by nitin-787** [comment]
@aashray374 cool, how are you going to approach this issue?

**Comment by aashray374** [comment]
I will use realtime database of firebase to implement the private chat feature.

**Comment by nitin-787** [comment]
@aashray374, last year I heard that Google changed something in Firebase. I'm afraid the backend might not still be functional, so we need to check that first."

**Comment by aashray374** [comment]
So we need to first verify that everything is still working as expected. We'll also need to identify and remove any deprecated services before proceeding further.

**Comment by nitin-787** [comment]
> So we need to first verify that everything is still working as expected. We'll also need to identify and remove any deprecated services before proceeding further.

indeed

**Comment by aashray374** [comment]
So I will first test the app before implementing the private chat feature.

**Comment by nitin-787** [comment]
> So I will first test the app before implementing the private chat feature.

yes you have to test it first, if it wont work then we'll have to fix the backend or in worst case will have to migrate backend to some other tech stack

**Comment by aashray374** [comment]
Got it! I'll start by testing the app. If the backend isn’t functioning properly, we'll troubleshoot and fix it. If necessary, we can explore migrating to another tech stack as a last resort.

**Comment by aashray374** [comment]
Sure, I'll start testing the app today. I'll ping you on Discord if I need any assistance. Also, could you please assign this task to me under Hacktoberfest?

**Comment by tanayduddalwar** [comment]
@nitin-787  can you assign me this issue

**Comment by nitin-787** [comment]
@tanayduddalwar did you read the above conversatation? if yes then ping me on discord, then link is the readme section
